packages/components/inputs/select-utils/src/custom-styled-select-options/optionStyle-checkbox-components.tsx

```diff
@@ -0,0 +1,66 @@
+import { type Props as ReactSelectProps } from 'react-select';
+import CheckboxInput from '@commercetools-uikit/checkbox-input';
+import { css } from '@emotion/react';
+import { designTokens } from '@commercetools-uikit/design-system';
+
+/**
+ * Returns custom components to be used with react-select, when optionStyle is set to "checkbox"
+ */
+export const optionStyleCheckboxComponents = () => {
+  return {
+    Option: (props) => {
+      const {
+        // children, - not needed, its just the "label" props content
```

Comment:
Not sure if this bunch of comments is helpful/needed. (I copied the list of available props from react-select docs to figure out what's possible and added some remarks).

Reply:
updated here: thank you